### ISSUE TITLE
Add vignette on modelling temporally aggregated data (i.e., `obs_opts(na = "accumulate")` )

### ISSUE THREAD

**Issue by jamesmbaazam** [epiforecasts/EpiNow2#772]
Thanks a lot all - I've tested it out and it's working well now! I've included the updated figure below on the same data for reference, in case helpful! 

@jamesmbaazam regarding extra documentation, I think that at some point it could be useful to add a short, dedicated section on applying to weekly data. The current documentation mentions the rw estimator for weekly, but I think having a section with an example as you do for daily could be beneficial to others as we work a lot with weekly aggregated data. The other thing would probably be then to be able to visualize on weekly scale too, but as it's possible for end users to extract the estimates and create visuals I don't think it's really necessary. 

Thanks again for your help and for a great package! 

<img width="505" alt="weekly_est_epinow" src="https://github.com/user-attachments/assets/77fe349e-caf2-4106-b4a3-1c3e677cb2b3">

_Originally posted by @martinamcm in https://github.com/epiforecasts/EpiNow2/issues/767#issuecomment-2351691083_

**Comment by jamesmbaazam** [comment]
By "surface", I mean we need a vignette walkthrough. My argument for this need is here https://github.com/epiforecasts/EpiNow2/pull/774#discussion_r1760886569.

**Comment by seabbs** [comment]
Agree. Perhaps edit the title and add a plan/outline?

**Comment by jamesmbaazam** [comment]
# Outline of vignette

- Discuss how EpiNow2 treats/prepares input data (into complete daily data)
- Motivate for data being reported at different types of intervals;
	- daily, 
	- irregular intervals (every few days), and
	- regular but at an aggregated scale (weekly, monthly)
- Walk-throughs of how EpiNow2 supports observations reported at non-daily intervals:
   - Treating missing dates as explicitly missing (`obs_opts(na = "missing")`)
   - Treating missing dates as accumulated (`obs_opts(na = "accumulate")`)

**Comment by seabbs** [comment]
We might want to wait on this until we have some sort of clarity on the approach that is going to be used to deal with https://github.com/epiforecasts/EpiNow2/issues/547?

**Comment by seabbs** [comment]
Sure but if any change is imminent that might be more work? On the other hand if we still need to discuss what #547 looks like it could help inform that discussion.

**Comment by jamesmbaazam** [comment]
Agrees. Let's hold this off for now. The question about treatment of missingness has been coming up a lot lately.

**Comment by seabbs** [comment]
This can be done as part of the manuscript that @jamesmbaazam  is writing with @pearsonca as a nice open source contribution? I am imagining basically pulling out a single fit from that paper, working through the steps simply, visualsing and saying wow so different and then linking out to the manuscript for people wanting more.

**Comment by pearsonca** [comment]
> This can be done as part of the manuscript that [@jamesmbaazam](https://github.com/jamesmbaazam) is writing with [@pearsonca](https://github.com/pearsonca) as a nice open source contribution? I am imagining basically pulling out a single fit from that paper, working through the steps simply, visualsing and saying wow so different and then linking out to the manuscript for people wanting more.

makes sense to me - i'd suggest one modification: use some sort data set already associated with EpiNow2 vignettes? i suspect we'd be able to get the same "wow", and makes it a bit more focused example experience (rather than having a bunch of code that grabs SA data, parses it particular way, etc, etc).

**Comment by seabbs** [comment]
yup good point

**Comment by pearsonca** [comment]
welp, i'm happy to bang out a draft PR that incorporates the distinct pipeline elements by copy-pasting those into the shell of some other current vignette here. can't commit to polishing it to fruition, however.

if someone else will volunteer for finishing said PR, i'm happy to do quick turn on the scut work.

**Comment by seabbs** [comment]
@jamesmbaazam ?

**Comment by jamesmbaazam** [comment]
Sounds good to me but I am currently...swamped!! PR can be submitted then we can decide on scope and steps to push to finishing line. 

Also, @sbfnk You were more convinced this should rather be a blog post on the feature release? 👀

**Comment by sbfnk** [comment]
I don't feel strongly either way, but a blog post would be less effort and could use existing code.

**Comment by jamesmbaazam** [comment]
Linking to this work I'm doing here https://github.com/epiverse-trace/epiverse_demos/pull/2 for the HPRU event. We could expand on it later.
